Marlin/Marlin_main.cpp

```diff
@@ -263,6 +263,7 @@ static millis_t max_inactive_time = 0;
 static millis_t stepper_inactive_time = DEFAULT_STEPPER_DEACTIVE_TIME * 1000L;
 millis_t print_job_start_ms = 0; ///< Print job start time
 millis_t print_job_stop_ms = 0;  ///< Print job stop time
+static millis_t last_command_time = 0;
```

Comment:
Does this endless print wait every second when there is no serial activity?
There was a PR to do this with OK that was rejected because the host should
time out waiting for OK and send the command again.

On 16 April 2015 at 14:59, Scott Lahteine <notifications@github.com> wrote:

> In Marlin/Marlin_main.cpp
> <https://github.com/MarlinFirmware/Marlin/pull/1922#discussion_r28511493>:
>
> > @@ -263,6 +263,7 @@ static millis_t max_inactive_time = 0;
> >  static millis_t stepper_inactive_time = DEFAULT_STEPPER_DEACTIVE_TIME * 1000L;
> >  millis_t print_job_start_ms = 0; ///< Print job start time
> >  millis_t print_job_stop_ms = 0;  ///< Print job stop time
> > +static millis_t last_command_time = 0;
>
> If this is only going to be used in get_command, you can put this static
> inside that function.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/MarlinFirmware/Marlin/pull/1922/files#r28511493>.
>